manifests/server/grant_role.pp

```diff
@@ -0,0 +1,43 @@
+# Define for granting membership to a role. See README.md for more information
+define postgresql::server::grant_role (
+  $group,
```

Comment:
Thanks for the contribution @janfabry! Can you add some validation for `$group` and `$role` so that they don't get passed in as undef or invalid values? Looks like if there are invalid values passed into either of these parameters, then we will continue to execute a bad psql command.

Thanks!